src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/DatabaseTreeNode.cs

```diff
@@ -79,8 +79,10 @@ public bool IsAccessible(SmoQueryContext context)
             }
             catch (Exception ex)
             {
-                // IsAccessible is not set of DW Gen3 so exception is expected in this case
-                if (IsDWGen3(context?.Database))
+                // IsAccessible is not set of DW Gen3 and Dataverse so exception is expected in this case
+                // Incase of dataverses and other TDS endpoints isAccessible creates a temp table to check if the database  
+                // is accessible, however these endpoints may not support ddl statements and therefore the check fails.
+                if (IsDWGen3(context?.Database) || isUnknownDatabaseEdition(context?.Database))
```

Comment:
If this Enum starts including DataVerse in future, we might start getting issues.. as it won't be DW Gen3 and neither an unknown.. is there no other identification available for identifying whether `db` isDataVerse?

cc @shueybubbles do you have any thoughts?

Reply:
Since Dataverse is officially supported by SSMS ([MS Docs](https://learn.microsoft.com/en-us/power-apps/developer/data-platform/dataverse-sql-query)), I wonder if we could do better to identify dataverse database from SMO.

Reply:
uh I don't think our team stated SSMS supports it, their team is saying it will work with SSMS.